lua/fzf.lua

```diff
@@ -111,17 +162,17 @@ function FZF.raw_fzf(contents, fzf_cli_args)
           if done_state then return end
           if usrval == nil then
             on_done()
-            cb(nil)
+            if cb then cb(nil) end
             return
           end
           uv.fs_write(fd, tostring(usrval) .. "\n", -1, function (err, bytes)
             if err then
-              cb(err)
+              if cb then cb(err) end
               on_done()
               return
             end
 
-            cb(nil)
+            if cb then cb(nil) end
```

Comment:
I don't think I want the user to ignore the error in all cases, only when it no error is possible, which is when calling `cb(nil)`.

I know that this makes it a bit annoying to program for very quick scripts, but I think the defaults should encourage good practices.

Reply:
Makes sense,  in such case we can call that “an information error”.